src/picongpu/include/simulation_defines/param/fileOutput.param

```diff
@@ -63,6 +66,12 @@ namespace picongpu
             CreateEnergyDensityOperation<bmpl::_1>
             >::type EnergyDensity_Seq;
 
+    /* EnergyPerpendicular section: define longitudinal as 1=Y */
```

Comment:
actually it's not the ideal to copy & paste the documentation of that class, e.g. what its documented default is. I will add 0=x and 2=z